addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -581,6 +581,29 @@ def _l10n_in_edi_generate_invoice_json(self, invoice):
                 json_payload["ExpDtls"].update({
                     "Port": invoice.l10n_in_shipping_port_code_id.code
                 })
+            total_json = json_payload['ValDtls']
+            base_and_tax_amount = tax_details.get("base_amount") + tax_details.get("tax_amount")
+            # For Export If with payment of Tax then we need to include Tax in Total Invoice Value
+            if json_payload['TranDtls']['SupTyp'] == 'EXPWP' and total_json['AssVal'] == base_and_tax_amount:
```

Comment:
I'm not a fan of preparing the JSON, and then updating the values inside based on the values inside...
but this seems to be what was done before 🙄